src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -92,7 +99,17 @@ public PowerShellProcessInstance(Version powerShellVersion, PSCredential credent
 
         /// <summary>
         /// </summary>
-        public PowerShellProcessInstance() : this(null, null, null, false)
+        /// <param name="powerShellVersion"></param>
+        /// <param name="credential"></param>
+        /// <param name="initializationScript"></param>
+        /// <param name="useWow64"></param>
+        public PowerShellProcessInstance(Version powerShellVersion, PSCredential credential, ScriptBlock initializationScript, bool useWow64) : this(powerShellVersion, credential, initializationScript, useWow64, null)
+        {
+        }
+
+        /// <summary>
+        /// </summary>
```

Comment:
Please add some summary information here

Reply:
I added some information here. If you feel that more information is needed or have a suggestion for the text let me know and I will improve it :)